since go 1.22, CGO_ENABLED=0 and -buildmode=pie does not play for several
architectures, in our case: armhf, armv7, loongarch64, riscv64, s390x & x86

--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,6 @@ ifeq ($(UNAME_S),Darwin)
 	BUILD_CGO_ENABLED ?= 1
 else
 	shasum = sha512sum
-	BUILD_CGO_ENABLED ?= 0
 endif
 
 .PHONY: all
